tests/network/general/test_ip_family_services.py

```
@@ -0,0 +1,203 @@
+SERVICE_IP_FAMILY_POLICY_REQUIRE_DUAL_STACK = "RequireDualStack"
+
+
+def assert_svc_ip_params(
+    svc,
+    expected_num_families_in_service,
+    expected_ip_family_policy,
+):
+    assert (
+        len(svc.instance.spec.ipFamilies) == expected_num_families_in_service
+        and svc.instance.spec.ipFamilyPolicy == expected_ip_family_policy
+    ), f"{expected_ip_family_policy} service wrongly created."
```

Ruff (0.14.13)

Assertion should be broken down into multiple parts

(PT018)